lib/list_versions_test.go

```diff
@@ -9,31 +9,30 @@ import (
 )
 
 const (
-	hashiURL = "https://releases.hashicorp.com/terraform/"
+	hashiURL = "https://api.releases.hashicorp.com/v1/releases/terraform"
 )
 
 // TestGetTFList : Get list from hashicorp
-func TestGetTFList(t *testing.T) {
 
-	listAll := true
-	list, _ := lib.GetTFList(hashiURL, listAll)
+func TestGetTFReleases(t *testing.T) {
```

Comment:
My [last commit](https://github.com/warrensbox/terraform-switcher/pull/227/commits/1adaf20372c64cf17c177fd038ddbeca949eff68) broke this test case so it needs reworked. Will work on this later tonight.